Sources/TuistGenerator/Utils/EmbedScriptGenerator.swift

```diff
@@ -65,17 +71,21 @@ final class EmbedScriptGenerator: EmbedScriptGenerating {
             if let dsymPath = frameworkMetadataProvider.dsymPath(frameworkPath: frameworkPath) {
                 let relativeDsymPath = dsymPath.relative(to: sourceRootPath)
                 script.append("install_dsym \"\(relativeDsymPath.pathString)\"\n")
-                inputPaths.append(relativeDsymPath)
-                outputPaths.append("${DWARF_DSYM_FOLDER_PATH}/\(dsymPath.basename)")
+                if includeSymbolsInFileLists {
```

Comment:
@paulsamuels would you mind adding a brief comment here explaining why this is necessary. It'll be useful for people coming across this code in the future and trying to understand why we need this. Maybe a reference to the original issue too?